src/modules/select/index.ts

```diff
@@ -13,7 +13,8 @@ const defaultWalletNames = [
   'opera',
   'operaTouch',
   'status',
-  'unilogin'
+  'unilogin',
+  'MYKEY'
```

Comment:
```suggestion
  'mykey'
```
Please make this Lowercase or camelCase to match other wallet names